perception/sub8_perception/nodes/path_marker_finder.py

```diff
@@ -37,6 +38,11 @@ class PathMarkerFinder():
     # Model of four corners of path marker, centered around 0 in meters
     LENGTH = 0.6096 # Longer side of path marker in meters
     WIDTH = 0.0762 # Shorter Side
+
+    # Demensions of the prop we made for testing
```

Comment:
It would be pretty tough to make it part of the service request, as the pose estimations are done continuously as frames come in not when the pose is requested. It could be a rosparam, but I prefer it hardcoded so I have to be very intensional about changing it (see a diff in a source file).